socorro/processor/processor_2015.py

```diff
@@ -115,9 +113,9 @@ class Processor2015(RequiredConfig):
         reference_value_from='resource.postgresql',
     )
     required_config.add_option(
-        'version_string_api',
-        doc='url for the version string api endpoint',
-        default='https://crash-stats.mozilla.com/api/VersionString'
+        'buildhub_api',
+        doc='url for the Buildhub api',
+        default='https://buildhub.prod.mozaws.net/v1/'
```

Comment:
This should change to the full url. That way we can change the collection and bucket when testing more easily.